linkerd/app/core/src/admin/tasks.rs

```diff
@@ -0,0 +1,134 @@
+use futures::future;
+use http::{header, StatusCode};
+use hyper::{service::Service, Body, Request, Response};
+use std::task::{Context, Poll};
+use std::{
+    fmt::{self, Write},
+    io,
+};
+use tokio_trace::tasks::TaskList;
+#[derive(Clone)]
+pub struct Tasks {
```

Comment:
Not a blocker (obviously) but do you think this can live in hawkw/tokio-trace behind a feature flag?

Reply:
![PRs_welcome](https://user-images.githubusercontent.com/2796466/88345802-a24c7a00-ccfb-11ea-8b63-a201d93ce622.jpg)